<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Users</class>
 <widget class="QDialog" name="Users">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пользователи</string>
  </property>
  <widget class="QLabel" name="L_SEARCH">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_SEARCH">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_SEARCH">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_USERS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пользователи</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_ADDUSER">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить пользователя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_LOGIN">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_PASSWORD">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_ROLE">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_ADDUSER">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_LOGIN">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_PASSWORD">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_ROLE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Роль</string>
   </property>
  </widget>
  <widget class="QTableView" name="TV_RESULT">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>401</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
